fili-core/src/main/java/com/yahoo/bard/webservice/async/ResponseException.java

```diff
@@ -64,7 +95,29 @@ public ResponseException(
             DruidQuery<?> druidQuery,
             Throwable cause
     ) {
-        super(buildMessage(reason, description, statusCode, druidQuery, cause));
+        this(statusCode, reason, description, druidQuery, cause, new ObjectMapper().writer());
+    }
```

Comment:
We should deprecate this constructor, since creating a new "naked" mapper w/o the extensions we have on the primary ones will result in a funky serialization.